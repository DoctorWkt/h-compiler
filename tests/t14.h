void puts(char *c)
{
  while (*c != '\0') {
    putchar(*c); c++;
  }
}

void main()
{
  char *x= "X now has a string attached\n";
  puts(x);
  x= x + 6;
  puts(x);
  x[12]='\n';
  x[13]='\0';
  puts(x);
}
