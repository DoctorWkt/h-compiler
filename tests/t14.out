X now has a string attached
has a string attached
has a string
